docs/src/misc.rst

```diff
@@ -273,6 +273,22 @@ API
 
     .. versionadded:: 1.6.0
 
+.. c:function:: int uv_os_tmpdir(char* buffer, size_t* size)
+
+    Gets the temp directory. On Windows, `uv_os_tmpdir()` uses `GetTempPathW()`.
+    On all other operating systems, `uv_os_tmpdir()` uses the first environment
+    variable found in the ordered list `TMPDIR`, `TMP`, `TEMP`, and `TEMPDIR`.
+    If none of these are found, the path `"/tmp"` is used, or, if the macro
+    `__ANDROID__` is defined, `"/data/local/tmp"` is used. The temp directory is
```

Comment:
maybe just say "on Android ..." there is no reason for that macro to be defined unless you're on Android.